src/graph/ExecutionEngine.cpp

```diff
@@ -29,7 +29,10 @@ Status ExecutionEngine::init(std::shared_ptr<folly::IOThreadPoolExecutor> ioExec
         return addrs.status();
     }
     metaClient_ = std::make_unique<meta::MetaClient>(ioExecutor, std::move(addrs.value()));
-    metaClient_->init();
+    if (!metaClient_->waitForMetadReady()) {
```

Comment:
the function is always return true, is this check really need?

Reply:
Now, because there is no clusterId, so this function aways return true. The next PR which support clusterId will change this code. Thanks!